migrations.json

```diff
@@ -0,0 +1,24 @@
+{
+  "generators": {
+    "update-plugin-name": {
+      "version": "1.1.0",
+      "description": "Change references to package from 'nx-python-pdm' to '@dman926/nx-python-pdm'",
```

Comment:
So nx-python-pdm is the nx default in this case?

Reply:
No this is just a migration to change the name. I created the project with `nx-python-pdm`, but npmjs wants a scope, so this migration adds it.

Reply:
Migrations are for modifying an existing workspace that is using the plugin. Their installations will default to `nx-python-pdm` if they create a project before this version, so the migration changes it when it is updated.